# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/glibc22/initfini-ccwrapper.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---


Make sure that the ROCK Linux CC Wrapper is not removing the CFLAGS
stored in CFLAGS-initfini.s and CFLAGS-pt-initfini.s.

--- ./csu/Makefile.orig	2001-07-23 20:53:37.000000000 +0300
+++ ./csu/Makefile	2002-06-23 09:25:47.000000000 +0300
@@ -98,5 +98,6 @@
 
 $(objpfx)initfini.s: initfini.c
-	$(compile.c) -S $(CFLAGS-initfini.s) -finhibit-size-directive \
+	CC_WRAPPER_APPEND="$$CC_WRAPPER_APPEND $(CFLAGS-initfini.s)" \
+		$(compile.c) -S -finhibit-size-directive \
 		$(patsubst -f%,-fno-%,$(exceptions)) -o $@
 
--- ./linuxthreads/Makefile.orig	2002-01-21 05:21:14.000000000 +0200
+++ ./linuxthreads/Makefile	2002-06-23 12:25:10.000000000 +0300
@@ -105,5 +105,6 @@
 
 $(objpfx)pt-initfini.s: pt-initfini.c
-	$(compile.c) -S $(CFLAGS-pt-initfini.s) -finhibit-size-directive \
+	CC_WRAPPER_APPEND="$$CC_WRAPPER_APPEND $(CFLAGS-pt-initfini.s)" \
+		$(compile.c) -S -finhibit-size-directive \
  		$(patsubst -f%,-fno-%,$(exceptions)) -o $@
 
